packages/wonder-blocks-data/util/no-cache.md

```diff
@@ -0,0 +1,55 @@
+The `ICache` interface is included below for reference in case you would like
+to implement your own caching strategy.
```

Comment:
The filename is `no-cache.md` but this file documents `ICache`, is this intentional?

Reply:
In short, yes, it is intentional.

This really documents `NoCache` mostly. It includes `ICache` so that it's in the docs. Just like the `RequestHandler` base class docs, which include the `IRequestHandler` interface.

Reply:
I see the confusion though. I forgot that code comments are only exported to docs for react components. I've updated the NoCache docs to accommodate this oversight. Thanks for catching it.

https://github.com/Khan/wonder-blocks/commit/259898380dd7222f26d100fe0af8131e2b559ae8